dispatcher/src/main/java/io/zeebe/dispatcher/Dispatcher.java

```diff
@@ -207,18 +216,22 @@ private long updatePublisherPosition(final int activePartitionId, final int newO
     if (newOffset > 0) {
       newPosition = position(activePartitionId, newOffset);
     } else if (newOffset == RESULT_PADDING_AT_END_OF_PARTITION) {
-      logBuffer.onActiveParitionFilled(activePartitionId);
+      logBuffer.onActivePartitionFilled(activePartitionId);
       newPosition = -2;
     }
 
+    if (publisherPosition.proposeMaxOrdered(newPosition)) {
```

Comment:
It was always necessary, I just moved it inside the function. The function is is called `updatePublisherPosition` so I thought it made sense to update the publisher position instead of us having to do that outside

Reply:
ok thanks